CHANGELOG.md

```diff
@@ -8,45 +8,30 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 ## Unreleased
 
+The following sections document changes that have been released already:
+
+## 1.5.1 - 2020-02-03
+
 ### Deprecation
 
 #### browser
 
 - Deprecated SessionManager
-
-### Internal refactor
-
-These changes are being reflected here even though normally internal refactoring 
-shouldn't be mentioned in the CHANGELOG (since it's meant for the consumers of
-the library, whereas library developers would rather read the commit history).
-In this case though, it is indeed a change that should be mentioned to
-library users, as it's really a feature change: i.e., the implicit flow is no
-longer supported.
-
-Whether this is really breaking or not is up for debate, because while it
-technically is a breaking change, we don't think any OIDC issuers only support
+- The implicit flow is no longer supported. However, no OIDC issuer only supports
```

Comment:
I'm not sure, but is "OIDC issuer" too broad?

```suggestion
- The implicit flow is no longer supported. However, no known Solid server only supports
```

Reply:
That's a good point, it's more precise if we focus it to "Solid Identity Issuers" (Solid servers being a little unclear, since it may apply to both IdPs and Pod servers)